apps/mobile/src/components/ChatDetailScreen/components/QuickActionBanner.tsx

```diff
@@ -129,35 +129,36 @@ function QuickActionBanner(): JSX.Element | null {
     )
   }
 
-  if (
-    lastMessage.message.messageType === 'DELETE_CHAT' &&
-    lastMessage.state === 'received'
-  ) {
-    return (
-      <QuickActionBannerUi
-        topText={t('messages.messagePreviews.incoming.DELETE_CHAT', {
-          them: otherSideData.userName,
-        })}
-        bottomText={t('messages.leaveToo')}
-        headingType={'boldBottom'}
-        buttonText={t('messages.deleteChat')}
-        leftElement={
-          <UserAvatar
-            width={48}
-            height={48}
-            grayScale
-            userImage={otherSideData.image}
-          />
-        }
-        onButtonPress={() => {
-          Keyboard.dismiss()
-          void deleteChat().then((result) => {
-            if (result) resetNavigationToMessagingScreen()
-          })
-        }}
-      />
-    )
-  }
+  // if (
```

Comment:
Why leave it commented in here and not just delete it?

Reply:
Good point, removing

Reply:
Glad you asked. This one should have stayed 🥔